docs/kics-auto-remediation.md

```diff
@@ -0,0 +1,30 @@
+# KICS AUTO REMEDIATION
+
+With this new feature, KICS provides auto remediation for simple replacements and simple additions in a single line.
```

Comment:
auto remediation for simple replacements and simple additions in a single line 
---> 
auto remediation for single-line replacements and additions